charts/enterprise-logs/Makefile

```diff
@@ -0,0 +1,25 @@
+.ONESHELL:
+.DELETE_ON_ERROR:
+export SHELL     := bash
+export SHELLOPTS := pipefail:errexit
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rule
+
+# Adapted from https://suva.sh/posts/well-documented-makefiles/
+.PHONY: help
+help: ## Display this help
+help:
+	@awk 'BEGIN {FS = ": ##"; printf "Usage:\n  make <target>\n\nTargets:\n"} /^[a-zA-Z0-9_\.\-\/%]+: ##/ { printf "  %-45s %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+.PHONY: dep-update
+dep-update: ## Update chart dependencies
+dep-update:
+	helm dep update
+
+.PHONY: exports
```

Comment:
I have mixed feelings about my introduction of exports to the repository. It's useful to see the manifestation of a change but it really muddies all Pull Requests, especially since a version bump is mandated and causes every manifest to change in at least that respect.

I think I would rather leave this out of the GEL chart and reintroduce it as some kind of CI or manual job with a script that filters the dull differences.